rakelib/artifacts.rake

```diff
@@ -43,7 +43,17 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
-  task "prepare" => ["bootstrap", "plugin:install-default"]
+  task "use-defaults-gemfile" do
+    FileUtils.cp("Gemfile.defaults", "Gemfile")
```

Comment:
I might be missing context, so bear with me - This file copy feels... weird. Is the ~/Gemfile not important anymore?